dolphinscheduler-server/src/main/resources/logback-worker.xml

```diff
@@ -56,10 +56,9 @@
     </appender>
     <appender name="WORKERLOGFILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>${log.base}/dolphinscheduler-worker.log</file>
-        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+        <!-- <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
             <level>INFO</level>
-        </filter>
-        <filter class="org.apache.dolphinscheduler.server.log.WorkerLogFilter"/>
```

Comment:
I still have a question that is it a good way to remove `WorkerLogFilter`. Removing `WorkerLogFilter` directly makes `WorkerLogFilter` unavailable, so can we modify `WorkerLogFilter`  to achieved the goal?
```
public FilterReply decide(ILoggingEvent event) {
    if (event.getThreadName().startsWith("Worker-")){
        return FilterReply.ACCEPT;
    }

    return FilterReply.DENY;
}
```